src/data/mod.rs

```diff
@@ -23,6 +23,18 @@ use crypto::{
 #[allow(needless_pass_by_value)]
 pub(crate) mod frame;
 
+lazy_static! {
+    static ref OPT_NET_ID: I2PString = "netId".into();
+    static ref OPT_ROUTER_VERSION: I2PString = "router.version".into();
+    static ref OPT_CAPS: I2PString = "caps".into();
+}
+
+lazy_static! {
+    pub static ref NET_ID: I2PString = "2".into();
+    static ref ROUTER_VERSION: I2PString = "0.9.37".into();
+    static ref CAPS: I2PString = "KU".into();
```

Comment:
This is a temporary fixed capability string, which will be replaced later with a dynamic one.

Reply:
Opened #41.